src/coordinator/ts/m3db/convert.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package m3db
+
+import (
+	"time"
+
+	"github.com/m3db/m3db/src/dbnode/encoding"
+	"github.com/m3db/m3db/src/dbnode/x/xio"
+	"github.com/m3db/m3x/ident"
+)
+
+const (
+	// todo(braskin): replace this with actual size if we can
+	blockReplicaLen = 10
+)
+
+// BlockReplica contains the replicas for a single m3db block
+type BlockReplica struct {
+	Start     time.Time
+	BlockSize time.Duration
+	Replicas  []encoding.MultiReaderIterator
+}
+
+// ConvertM3DBSeriesIterators takes in series iterators from m3db and returns
```

Comment:
nit: Something like `ConvertM3DBSeriesIterators converts m3db SeriesIterators to SeriesBlocks` might be more go-ey